src/smtsolvers/LookaheadSMTSolver.h

```diff
@@ -72,7 +72,7 @@ class LookaheadSMTSolver : public SimpSMTSolver {
     laresult expandTree(LANode & n, std::unique_ptr<LANode> c1,
                         std::unique_ptr<LANode> c2); // Do lookahead.  On success write the new children to c1 and c2
     std::unique_ptr<LookaheadScore> score;
-    bool okToPartition(Var v) const { return theory_handler.getTheory().okToPartition(theory_handler.varToTerm(v)); };
+    bool okToPartition(Var) const { return true; }
```

Comment:
It is always Ok to partition?
What's the point in this function then?
Mb makes sense to remove it completely?

Reply:
I think it may make sense to revisit this in the future (if someone will pick up parallel solving again).
I can imagine that it might make sense to not pick certain atoms for partitioning.
Currently, this was done only for the inequalities resulting from splitting arithmetic equalities.
I am not convinced these should be disallowed. Moreover, I would like to implement handling equalities and disequalities in LA engine directly, so there would be no more splitting equalities.

If I remove this function, I would have to update the partitioning algorithm as well, but I wanted to avoid that. I want to keep the changes to minimum.

Reply:
Ok, sounds legitimate